Makefile

```diff
@@ -313,11 +314,20 @@ host-plugin-release:
 	@echo dev: need docker login with user in contiv org
 	docker plugin push ${CONTIV_V2PLUGIN_NAME}
 
-only-tar:
-
-tar: clean-tar
-	CONTIV_NODES=1 ${MAKE} build
-	@tar -jcf $(TAR_FILE) -C $(GOPATH)/src/github.com/contiv/netplugin/bin netplugin netmaster netctl contivk8s netcontiv -C $(GOPATH)/src/github.com/contiv/netplugin/scripts contrib/completion/bash/netctl -C $(GOPATH)/src/github.com/contiv/netplugin/scripts get-contiv-diags
+##########################
+## Packaging and Releasing
+##########################
+
+# build tarball
+tar: compile-with-docker
+	docker run --rm netplugin-build:$(NETPLUGIN_CONTAINER_TAG) \
```

Comment:
Please use `docker cp` to copy build artifacts.

Reply:
there is no container to target docker cp

Reply:
switching to docker create + docker cp + docker rm